recipes/quicfire-tools/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "quicfire-tools" %}
+{% set version = "0.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/quicfire_tools-{{ version }}.tar.gz
+  sha256: c0c56fb12124651e3aaabab8409738a8d3e8039323a48459e94eaf359273faeb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
```

Comment:
Please have a look at the `noarch: python` docs:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python